receiver/awscloudwatchreceiver/logs.go

```diff
@@ -221,7 +225,11 @@ func (l *logsReceiver) pollForLogs(ctx context.Context, pc groupRequest, startTi
 }
 
 func (l *logsReceiver) processEvents(now pcommon.Timestamp, logGroupName string, output *cloudwatchlogs.FilterLogEventsOutput) plog.Logs {
+	var logRecord plog.LogRecord
 	logs := plog.NewLogs()
+
+	resourceMap := map[string](map[string]plog.ScopeLogs){}
```

Comment:
This solution sticks to one scope. Previously I used `map[string](map[string]plog.ResourceLogs)`, maybe that's clearer to show what's the intention, but more prone to bugs imo (because then we have to access the first scope each time).

Reply:
I think it would be clearer to manage a separate `ResourceLogs` for each resource, or to just work with separate `plog.LogRecordSlice`.

Reply:
I'll change that then, no problem.

Reply:
@djaglowski I changed that to `ResourceLogs`.